aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1473,7 +1473,24 @@ static at::Tensor _quantized_convolution_onednn(
   // Scales of ONEDNN and PyTorch are reciprocal
   const ideep::scale_t& src_scales = ideep::scale_t(1, 1.0 / act_scale);
 
-#if defined(IDEEP_VERSION_MAJOR) && IDEEP_VERSION_MAJOR>=3 && defined(IDEEP_VERSION_REVISION) && IDEEP_VERSION_REVISION == 0
+#if defined(IDEEP_VERSION_MAJOR) && IDEEP_PREREQ(3, 1, 0, 1)
```

Comment:
Since `IDEEP_PREREQ` uses not only `IDEEP_VERSION_MAJOR`, suggest to check all the needed macros. Maybe you can just check these macros inside `IDEEP_PREREQ`?